README.md

```diff
@@ -77,110 +77,87 @@ This accepts an array of the loggers' codes which will be used to select the log
 
 **1. Base Logger**
 
-The base logger only with the following format using a `message`, `action` and all remaining arguments are rendered in a JSON payload
+The base logger only with the following format using a `message`,
+`action` and all remaining arguments are rendered in a JSON payload
 
 ```ruby
 # Sample usage
 EventTracer.info action: 'Action', message: 'Message', other_args: 'data'
 => "[Action] message {\"other_args\":\"data\"}"
 ```
 
-**2. Appsignal**
+**2. Metrics**
 
-Appsignal 2.5.1 is currently supported for the following metric functions available for the EventTracer's log methods
+EventTracer allows sending metrics together with your log. Currently two
+monitoring services are supported: AppSignal and DataDog.
 
-- increment_counter
-- add_distribution_value
-- set_gauge
+All metrics are sent in `metrics` fields, for example:
 
-All other functions are exposed transparently to the underlying Appsignal class
-
-The interface for using the Appsignal wrapper is:
+```ruby
+EventTracer.info(
+  action: 'Action',
+  message: 'There is an action',
+  metrics: [
+    :metric_1,
+    { metric_2: :gauce },
+    { metric_3: { value: 10, type: :distribution } }
+  ]
+)
+```
 
-Key | Secondary key | Secondary key type | Values
---------------|-------------|------------------|-------
-appsignal | increment_counter | Hash | Hash of key-value pairs featuring the metric name and the counter value to send
-| | add_distribution_value | Hash | Hash of key-value pairs featuring the metric name and the distribution value to send
-| | set_gauge | Hash | Hash of key-value pairs featuring the metric name and the gauge value to send
+Extra data in the payload can also be filtered to create tags for each metric:
```

Comment:
what's the syntax like?

```
{ metric_1: { value: 1, tags: %w[xxx yyy] } }
```

sth like ☝️ ?

Reply:
`tags` would be filtered from the input. For example:

```ruby
datadog = EventTracer::DatadogLogger.new(Datadog::Statsd.new, allowed_tags: [:app, :tenant_id])
EventTracer.register :datadog, datadog

EventTracer.info(
  message: 'this is a message',
  action: 'SomeAction',
  metrics: [:metric_1, :metric_2],
  tenant_id: 'SomeTenant', # this will be filtered as tags for metrics above
  other_data: 'other_data'
)
```